libnetdata/socket/socket.h

```diff
@@ -49,13 +49,23 @@ typedef struct listen_sockets {
     WEB_CLIENT_ACL fds_acl_flags[MAX_LISTEN_FDS];  // the acl to apply to the open sockets (dashboard, badges, streaming, netdata.conf, management)
 } LISTEN_SOCKETS;
 
+struct rrdpush_destinations {
+    char destination[100 + 1];
```

Comment:
I suppose that you used `100 +1` instead `101`, because this `100` is a specific definition, please, can you add a `define` for it instead to use `100`?

Reply:
Hi @thiagom, thanks, will change it but also will do some some changes and remove from socket.h (not the best place for this).

Reply:
Done.